src/py42/clients/securitydata.py

```diff
@@ -468,3 +469,18 @@ def destination_guid(self):
     def node_guid(self):
         """The GUID of the storage node containing the archive."""
         return self._node_guid
+
+
+def _escape_quote_chars_in_token(token):
+    """
+    The `nextPgToken` returned in Forensic Search requests with > 10k results is the eventId
+    of the last event returned in the response. Some eventIds have double-quote chars in
+    them, which need to be escaped when passing the token in the next search request.
+    """
+    unescaped_quote_pattern = r'[^\\]"'
+
+    return re.sub(
+        pattern=unescaped_quote_pattern,
+        repl=lambda match: match.group().replace(u'"', r'\"'),
```

Comment:
Do we need to worry about escaped-single quotes at all?

Reply:
Probably not until we actually find `eventIds` with single-quotes. I think the double-quotes being in some IDs is likely a bug on the backend.